### PR TITLE
Add -s to synopsis message to grdmask

### PR DESCRIPTION
**Pull request by srijac**
**Description of proposed changes**

This PR adds -s to grdmask.

Addresses #5653 

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Review by PaulWessel — COMMENTED** [review]
Thanks, but I think you placed the SYN_OPT-s too far down, it should fit in alphabetically.  Can you update it?

**Comment by PaulWessel** [comment]
Let us know if you are stuck and need help - it can be later in a break or even after the workshop.

File: doc/rst/source/grdmask.rst [hunk 1]
```diff
@@ -29,11 +29,11 @@ Synopsis
 [ |SYN_OPT-n| ]
 [ |SYN_OPT-qi| ]
 [ |SYN_OPT-r| ]
+[ |SYN_OPT-s| ]
 [ |SYN_OPT-w| ]
 [ |SYN_OPT-x| ]
 [ |SYN_OPT-:| ]
 [ |SYN_OPT--| ]
```
Comment (weiji14) [inline doc/rst/source/grdmask.rst:36]:
```suggestion
[ |SYN_OPT--| ]

```

**Review by weiji14 — APPROVED** [review]

**Review by meghanrjones — APPROVED** [review]
Great work, thanks @srijac!

**Pull request merged**